pkgs/tools/security/polkit-gnome/default.nix

```
@@ -20,10 +17,10 @@ in stdenv.mkDerivation rec {
   postInstall = ''
     mkdir -p $out/etc/xdg/autostart
     substituteAll ${./polkit-gnome-authentication-agent-1.desktop} $out/etc/xdg/autostart/polkit-gnome-authentication-agent-1.desktop
-    '';
+  '';
 
   meta = {
-    homepage = https://hal.freedesktop.org/docs/PolicyKit/;
+    homepage = "https://gitlab.gnome.org/Archive/policykit-gnome";
     description = "A dbus session bus service that is used to bring up authentication dialogs";
     license = stdenv.lib.licenses.gpl2;
```

```suggestion
    license = stdenv.lib.licenses.lgpl2Plus;
```

https://gitlab.gnome.org/Archive/policykit-gnome/blob/a0763a246a81188f60b0f9810143e49224dc752f/src/main.c#L5-7

Unfortunately, some of the older packages were not checked very rigorously.